jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -315,28 +315,48 @@ def uncert_cube(self):
     def spectral_display_unit(self):
         return astropy.units.Unit(self.app._get_display_unit(self.slice_display_unit_name))
 
+    @property
+    def mask_non_science(self):
+        # Aperture masks begin b removing from consideration any pixel
```

Comment:
nitpick, but since I noticed... 
```suggestion
        # Aperture masks begin by removing from consideration any pixel
```